cypress/integration/basic/person_modal.spec.js

```diff
@@ -0,0 +1,28 @@
+describe('Test that the person show modal can be opened', () => {
+  const BOOKMARKED_PERSON = 'Sadaf Arshad';
+
+  it('Bookmark a person', () => {
+    cy.visit('/applications');
+
+    cy.get('input#search_sidebar.input-large.search-query')
+    .type(BOOKMARKED_PERSON, {delay: 100})
+    .should('have.value', BOOKMARKED_PERSON)
+    .type('{enter}');
+  });
+
+  it('Clicking a bookmarked person\'s open icon should reveal a modal',() => {
```

Comment:
When you want to use a single quote within a single quoted string, change the string itself to be double-quoted, e.g.

"Clicking a bookmarked person's open icon should reveal a modal." (note the double quoted strings on the outside)